crates/matrix-sdk/src/room/mod.rs

```diff
@@ -1732,12 +1739,14 @@ impl Room {
     #[instrument(skip_all)]
     pub fn send_attachment<'a>(
         &'a self,
-        body: &'a str,
+        caption: Option<String>,
+        formatted: Option<FormattedBody>,
```

Comment:
Usually optional parameters come last.

Also `formatted` should probably be named `formatted_caption`, to make it clearer what it is.